src/SupportMessage/SupportMessage.stories.tsx

```diff
@@ -28,9 +29,9 @@ WithCustomDescription.args = {
   type: 'info',
   title: 'A SupportMessage using the Link component',
   description: (
-    <>
+    <Box>
```

Comment:
there doesn't seem to be any benefit of importing `Box` so it might be worth just using a `div` 😄